src/test/java/com/github/fppt/jedismock/TestCommandParser.java

```diff
@@ -63,23 +68,23 @@ public void testConsumeCount2() {
     @Test
     public void testConsumeParameter() throws ParseErrorException {
         InputStream stream = new ByteArrayInputStream("$5\r\nabcde\r\n".getBytes());
-        assertEquals(SliceParser.consumeParameter(stream).toString(), "abcde");
+        assertThat(consumeParameter(stream).toString()).isEqualTo("abcde");
     }
 
     @Test
     public void testParse() throws ParseErrorException {
         RedisCommand cmd = RedisCommandParser.parse("*3\r\n$0\r\n\r\n$4\r\nabcd\r\n$2\r\nef\r\n");
-        assertEquals(cmd.parameters().get(0).toString(), "");
-        assertEquals(cmd.parameters().get(1).toString(), "abcd");
-        assertEquals(cmd.parameters().get(2).toString(), "ef");
+        assertThat(cmd.parameters().get(0).toString()).isEqualTo("");
+        assertThat(cmd.parameters().get(1).toString()).isEqualTo("abcd");
+        assertThat(cmd.parameters().get(2).toString()).isEqualTo("ef");
     }
 
     @Test
     public void testConsumeCharacterError() throws ParseErrorException {
         InputStream stream = new ByteArrayInputStream("".getBytes());
         try {
```

Comment:
Here and in all the similar places it should be `assertThatThrownBy`

Reply:
Fixed